docs/Using-Fleet/Mobile-device-management.md

```diff
@@ -2,14 +2,58 @@
 
 MDM features are not ready for production and are currently in development. These features are disabled by default.
 
-MDM features allow you to mange macOS updates and macOS settings on your hosts.
+MDM features allow you to manage macOS updates and macOS settings on your hosts.
 
-To use MDM features you have to connect Fleet to Apple Push Certificates Portal:
+## Apple Push Notification Service (APNs)
```

Comment:
```suggestion
To use MDM features you have to connect Fleet to Apple Push Certificates Portal. See how [here](#apple-push-notification-service-apns).

## Apple Push Notification Service (APNs)
```

Hey, can we keep the "To use MDM features..." one line summary? This way the top of the page tells the user "you have to do this to use these features."

Then we can link them to the APNs section.

Also, I think we should move APNs and ABM sections to the bottom of the page. Even though they're the first step in setting up MDM features, the user will only follow these instructions once (and 1 year later when they renew). The user is likely to come back to this page to read about the OS updates (macOS updates) and Profiles (macOS settings) features more frequently. What do you think?

Reply:
:+1: sounds good, I'll update the PR.